content/02_general-rules/04_asset-modifications.md

```diff
@@ -1,10 +1,63 @@
 ### Asset Modifications
 
-Texture modifications are allowed, with some caveats. The textures of portal surfaces
-must not be modified in any way. No textures can be modified to provide any visual
-assistance, such as help with a lineup. No texture may be modified to be more
-translucent than it was, with the exception of the player's viewmodel.
-
-Modifying models is not allowed, with the exception of the portal gun (including both
-the viewmodel and the in-world model). Modifying audio files, captions or any other
-asset is not allowed.
+The asset modification rules will be split into their respective categories that are 
```

Comment:
Don't describe how the rules document is laid out here, people can already see that: just say something like "We have strict rules regarding which assets are allowed to be changed from their defaults during runs, which (unless otherwise specified) are shared across all games and categories. These rules vary depending on the kind of asset being modified." and go into the sections